vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -348,6 +348,35 @@ public String getCodeGrantAuthorizationURI() {
     }
 
     /**
+     * Authorizes users of the app who are not signed in. This call requires a client id and client secret
+     * to be set on the initial Configuration.
+     * <p/>
+     * Leaves User as null in {@link VimeoAccount} model and populates the rest.
+     * <p/>
+     * WARNING: This contains a synchronous network call. Use {@link #authorizeWithClientCredentialsGrant(AuthCallback)}
+     * for asynchronous use.
+     */
+    public VimeoAccount authorizeWithClientCredentialsGrantSync() {
+
+        Call<VimeoAccount> call = mVimeoService.authorizeWithClientCredentialsGrant(getBasicAuthHeader(),
+                                                                                    Vimeo.CLIENT_CREDENTIALS_GRANT_TYPE,
+                                                                                    mConfiguration.scope);
+
+        VimeoAccount vimeoAccount = null;
+        try {
+            retrofit2.Response<VimeoAccount> response = call.execute();
+             if (response.isSuccessful()) {
+                vimeoAccount = response.body();
+                saveAccount(vimeoAccount, null);
+            }
+        } catch (IOException e) {
+            ClientLogger.e("Exception during authorizeWithClientCredentialsGrantSync: " + e.getMessage(), e);
+        }
+
+        return vimeoAccount;
```

Comment:
I would suggest annotating this method nullable since it'll return null in the event of an IOException. Or have the method throw an exception - I think either would get the point across.